packages/squiggle-lang/__tests__/Distributions/Dotwise_test.res

```diff
@@ -0,0 +1,50 @@
+open Jest
+open Expect
+open TestHelpers
+open FastCheck
+open Arbitrary
+open Property.Sync
+
+describe("dotSubtract", () => {
+  test("mean of normal minus exponential (unit)", () => {
+    let mean = 0.0
+    let rate = 10.0
+
+    let dotDifference = DistributionOperation.Constructors.pointwiseSubtract(
+      ~env,
+      mkNormal(mean, 1.0),
+      mkExponential(rate),
+    )
+    let meanResult = E.R2.bind(DistributionOperation.Constructors.mean(~env), dotDifference)
+    let meanAnalytical = mean -. 1.0 /. rate
+    switch meanResult {
+    | Ok(meanValue) => meanValue->expect->toBeCloseTo(meanAnalytical)
+    | Error(err) => err->expect->toBe(DistributionTypes.OperationError(DivisionByZeroError))
```

Comment:
Shouldn't we just expect it to be one error for this one test?

Reply:
in next push it raises when it goes down any error path at all.